integration_tests/mockApis/govukOneLogin.ts

```diff
@@ -41,8 +42,7 @@ const redirect = () =>
         scope: { equalTo: 'openid email phone' },
         client_id: { equalTo: 'clientId' },
         state: { matches: '.*' },
-        redirect_uri: { equalTo: 'http://localhost:3007/auth/callback' },
-        nonce: { matches: '.*' },
```

Comment:
Remove match on `nonce` as it is not always passed (and other tests check that when it is it is used correctly).